gecode/int.hh

```diff
@@ -5172,6 +5173,58 @@ namespace Gecode {
          const Symmetries& syms,
          BoolBranchFilter bf=nullptr,
          BoolVarValPrint vvp=nullptr);
+
+#ifdef GECODE_HAS_CBS
+
+  /**
+   * \brief Branch over \a x using counting-based search
+   *
+   * Branches on the <variable, value> pair that has the the highest solution
+   * density across all active propagators. Computing solution density is
+   * currently supported for the following propagators:
+   *
+   *   - Gecode::Int::Distinct::Val
+   *   - Gecode::Int::Distinct::Bnd
+   *   - Gecode::Int::Distinct::Dom
+   *   - More to come...
+   *
+   * If the space does not contain any of the preceding propagators, this
+   * brancher won't be able to make any branching choices. For more details,
+   * please see the documentation in MPG, section ?.
```

Comment:
I will make a pull request to add documentation in MPG.

Reply:
If you want, I can give you access to MPG. It is slightly complicated, though.

Reply:
If you want, I can give you access to MPG. It is a little scary, though...